### ISSUE TITLE
Update for COPYING (last date 2007)?

### ISSUE THREAD

**Issue by GitMensch** [westes/flex#485]
... just wondering ...

**Comment by westes** [comment]
You're quite correct that COPYING and any other copyright notices need updated. I have a vague memory that some automated tool exists to do this, though.

**Comment by GitMensch** [comment]
Yes, there are automated tools, for example [gnulib's update-copyright](https://git.savannah.gnu.org/cgit/gnulib.git/tree/build-aux/update-copyright) or [Bison's update-package-copyright-year](https://github.com/akimd/bison/blob/master/build-aux/update-package-copyright-year), but those cannot be post-applied.
From my understanding the years should be updated when copyrightable changes were done, COPYING is a common file so we can just inspect the commit logs if every year has changes, then change that to -2021.

The same can be done for the library version (if that is printed/returned somewhere) and the output of `flex --version`.

For other copyright notices I'd say that should be investigated per file (again with the help of the git logs), a possibly good option is to run `LANG=en git log --follow $file | grep Date` - but the "per file" part could also happen after a release (because the last one hadn't those cleaned up either).

**Comment by GitMensch** [comment]
Friendly ping @westes.
